lux-lib/src/build/mod.rs

```diff
@@ -46,7 +46,7 @@ pub mod variables;
 /// over how a package should be built.
 #[derive(Builder)]
 #[builder(start_fn = new, finish_fn(name = _build, vis = ""))]
-pub struct Build<'a, R: Rockspec + HasIntegrity> {
+pub struct Build<'a, R: RemoteRockspec + HasIntegrity> {
```

Comment:
question: wouldn't we want to be able to build both local and remote rockspecs?
I suppose this would require local rockspecs to be able to provide the path to the project.

Reply:
Anything that's a RemoteRockspec has to be a LocalRockspec already, so we'll maybe have to add a special `.local()` function to the builder to differentiate? That's for a different pull request for sure :)

Reply:
oh I see what you mean, you want to be able to pass just a localrockspec. We might need a special refactor for this case indeed.